doc/man7/life_cycle-mac.pod

```diff
@@ -99,10 +99,10 @@ This is the canonical list.
  EVP_MAC_update                                     updated     updated
  EVP_MAC_final                                                  finaled
  EVP_MAC_CTX_free                freed   freed       freed       freed       freed
- EVP_MAC_gettable_ctx_params             newed    initialised   updated    
- EVP_MAC_settable_ctx_params             newed    initialised   updated
- EVP_MAC_CTX_gettable_params             newed    initialised   updated
- EVP_MAC_CTX_settable_params             newed    initialised   updated
+ EVP_MAC_get_ctx_params                  newed    initialised   updated    
+ EVP_MAC_set_ctx_params                  newed    initialised   updated
+ EVP_MAC_CTX_get_params                  newed    initialised   updated
+ EVP_MAC_CTX_set_params                  newed    initialised   updated
```

Comment:
This does not look correct either. The original is wrong but the new is wrong either. There should be these 4 functions EVP_MAC_CTX_get/set_params and EVP_MAC_CTX_gettable/settable_params.